triton/stream.py

```diff
@@ -7,6 +7,10 @@
 from boto.kinesis.exceptions import ProvisionedThroughputExceededException
 import boto.regioninfo
 
+import boto
+boto.config.add_section('Boto')
+boto.config.set('Boto', 'http_socket_timeout', '3')
```

Comment:
I think some comments explaining why this is might be nice.

Also, is it possible to check if the config value has already been explicitly set? Probably not very nice to override someone else's settings implicitly by importing a module.

Actually, this whole thing makes me a bit nervous. Maybe it's better to set something like this in the application rather than in this library?